apps/codestyle/codestyle.py

```diff
@@ -0,0 +1,148 @@
+import os
+import sys
+
+# Get the src directory of the project
+src_directory = os.path.join(os.getcwd(), 'src')
+
+# Global variables
+error_handler = False
+results = {
+    "Multiple blank lines check": "Passed",
+    "Trailing whitespace check": "Passed",
+    "GetCounter() check": "Passed",
+    "NpcFlagHelpers check": "Passed"
+}
+
+# Main function to parse all the files of the project
+def parsing_file(directory):
+    for root, _, files in os.walk(directory):
+        for file in files:
+            if not file.endswith('.ico'):                                   # Skip .ico files that cannot be read
+                file_path = os.path.join(root, file)
+                file_name = file
+                try:
+                    with open(file_path, 'r', encoding='utf-8') as file:
+                        multipleBlankLines_check(file, file_path)
+                        trailing_whitespace_check(file, file_path)
+                        getCounter_check(file, file_path)
+                        if file_name != 'Unit.h':
+                            npcFlagHelpers_check(file, file_path)
+                except UnicodeDecodeError:
+                    print(f"\nCould not decode file {file_path}")
+                    sys.exit(1)
+    # Output the results
+    print("")
+    for check, result in results.items():
+        print(f"{check} : {result}")
+    if error_handler:
+        print("\nPlease fix the codestyle issues above.")
+        sys.exit(1)
+    else:
+        print(f"\nEverything looks good")
+
+# Codestyle patterns checking for multiple blank lines
+def multipleBlankLines_check(file, file_path):
```

Comment:
I would respect the python snake_case naming here and moreover, let's add the type hints everywhere in any function header please
```suggestion
def multiple_blank_lines_check(file: str, file_path: str) -> None:
```

Reply:
file is not really a string but a full text. Are you sure about this type hints ? Pycharm return me a warning and suggest to define file as: TextIO

Reply:
I guessed it, so may it's wrong, I would trust more Pycharm then hehe, the type depends on what os.walk() returns, and pycharm could get it from there, so should be correct TextIO